api/instance/v1/server_utils.go

```diff
@@ -132,18 +132,16 @@ func (s *API) GetServerType(req *GetServerTypeRequest) (*ServerType, error) {
 	return nil, errors.New("could not find server type %q", req.Name)
 }
 
-// GetServerUserDataRequest is used by GetServerType method.
+// GetServerUserDataRequest is used by GetServerUserData method.
 type GetServerUserDataRequest struct {
 	Zone     utils.Zone `json:"-"`
 	ServerID string     `json:"-"`
 
-	// Key define the user data key to get
+	// Key defines the user data key to get.
 	Key string `json:"-"`
 }
 
-// GetServerUserData get user data
-//
-// Get the content of a user data with the given key on a server
+// GetServerUserData gets the content of a user data on a server with the given key.
```

Comment:
`a user data` doesn't really work since `data` is plural. So either `user data` or either `a user data object`.

Reply:
> GetServerUserData gets the content of user data on a server for the given key.